src/Domain/RetailerDemListCreator.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Linn.DemStock.Domain
+{
+    using Linn.Common.Persistence;
+    using Linn.DemStock.Domain.Exceptions;
+    using Linn.DemStock.Domain.Repositories;
+    using Linn.DemStock.Domain.RetailerDemListActivities;
+
+    public class RetailerDemListCreator
```

Comment:
This class looks much better now but I feel like maybe it should exist at Facade level rather than within the domain...

@mr-sandy @richardiphillips any thoughts?

Reply:
Domain level. 
It's never going to be called from http and logic should be in the domain where possible.

Reply:
Fair play